Sources/AppcuesKit/Presentation/Traits/Appcues/AppcuesTooltipTrait.swift

```diff
@@ -38,15 +45,20 @@ internal class AppcuesTooltipTrait: StepDecoratingTrait, WrapperCreatingTrait, P
     func createWrapper(around containerController: ExperienceContainerViewController) throws -> UIViewController {
         let dialogWrapperViewController = ExperienceWrapperViewController<TooltipWrapperView>(wrapping: containerController)
         dialogWrapperViewController.configureStyle(tooltipStyle)
+        dialogWrapperViewController.bodyView.pointerSize = hidePointer ? nil : pointerSize
```

Comment:
Was `dialogWrapperViewController` maybe copied from something modal related? Wondering if there is a clearer name for this thing.

Reply:
Good point, `dialog` is wrong here.